src/core/src/package_managers/YumPackageManager.py

```diff
@@ -291,21 +285,21 @@ def is_package_version_installed(self, package_name, package_version):
         # Loaded plugins: product-id, search-disabled-repos, subscription-manager
         # Installed Packages
         # kernel.x86_64                                                                                   3.10.0-514.el7                                                                                    @anaconda/7.3
-        self.composite_logger.log_debug("\nCHECKING PACKAGE INSTALL STATUS FOR: " + str(package_name) + " (" + str(package_version) + ")")
+        self.composite_logger.log_verbose("[YPM] Checking package install status. [PackageName={0}][PackageVersion={1}]".format(package_name, package_version))
         cmd = self.single_package_check_installed.replace('<PACKAGE-NAME>', package_name)
         output = self.invoke_package_manager(cmd)
         packages, package_versions = self.extract_packages_and_versions_including_duplicates(output)
 
         for index, package in enumerate(packages):
             if package == package_name and (package_versions[index] == package_version):
-                self.composite_logger.log_debug(" - Installed version match found.")
+                self.composite_logger.log_debug("[YPM] > Installed version match found. [PackageName={0}][PackageVersion={1}]".format(package_name, package_version))
                 return True
             else:
-                self.composite_logger.log_debug(" - Did not match: " + package + " (" + package_versions[index] + ")")
+                self.composite_logger.log_verbose("[YPM] > Did not match: " + package + " (" + package_versions[index] + ")")
 
         # sometimes packages are removed entirely from the system during installation of other packages
         # so let's check that the package is still needed before
-
+        self.composite_logger.log_debug(" - Installed version match NOT found for: " + str(package_name) + "(" + str(package_version) + ")")
```

Comment:
nit: self.composite_logger.log_debug(" - Installed version match NOT found for: [PackageName={0}][PackageVersion={1}]".format(str(package_name), str(package_version)))

Reply:
Addressed.